src/overlays/actors/ovl_Dm_Stk/z_dm_stk.c

```diff
@@ -648,7 +648,7 @@ void DmStk_PlaySfxForPlayingWithFairiesCutscene(DmStk* this, PlayState* play) {
         case 72:
         case 77:
         case 79:
-            Actor_PlaySfxAtPos(&this->actor, NA_SE_PL_WALK_WATER2);
+            Actor_PlaySfxAtPos(&this->actor, NA_SE_PL_WALK_GROUND + SURFACE_SFX_OFFSET_TALL_GRASS);
```

Comment:
Just noting that it's still partially unresolved if it's worth splitting up sfx into their base + offset when applicable. Open to more thoughts on this